### DISCUSSION TITLE
Svelte support grateful

### DISCUSSION THREAD

**Discussion by 99bits** [GrapesJS/grapesjs#5403] [Ideas]
Thank you for making this awesome GrapeJs project. Svelte is extremely popular and is now on par with react and Vue.  It would be a great win-win situation for  both GrapeJs and Svelte.

**Comment by artf** [comment]
@99bits GrapesJS is framework agnostic, so you should be able to use it in any framework of choice

**Reply by 99bits** [reply]
That's great, I was thinking of of using GrapeJs within a Sveltekit app and also being able to use svelte custom components that has data coming in from a webservice api within GrapeJS.  Do you know if this is possible?  Also ar  there any examples of Svelte and GrapeJs.  Cheers
